### PR TITLE
My bio!

### PR DESCRIPTION
**Pull request by marcusmendes**
Test pull request.

### CODE REVIEW

**Comment by brianamarie** [comment]
@marcusmendes I see your file! I'm glad to see there's some bio information in it, too! 😄

**Comment by marcusmendes** [comment]
Hello, @brianamarie.

File: marcus_bio.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### Hometown
+- Rio de Janeiro
```
Comment (jsebastiantello) [inline marcus_bio.md]: You'll be inundated by people for the Olympic games soon. I hope you survive!

Comment (sktay) [inline marcus_bio.md]: @marcusmendes ... I'm no expert :wink: but looks good to me.

File: newfile.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+New file!
```
Comment (jdim) [inline newfile.txt]: Cool!

Comment (marcusmendes) [inline newfile.txt]: Thank you!

**Pull request closed without merging**